doc/source/methods/ALE.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<ul><li>missing a comma in $x = (x_1, ...x_d)$ before $x_d$?</li><li>missing a comma after _$d$ features_ ?</li><li>missing comma after <em>notation</em> in _To simplify notation_?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>